src/OrchardCore.Modules/OrchardCore.ContentLocalization/GraphQL/LocalizationQueryObjectType.cs

```diff
@@ -13,6 +18,27 @@ public LocalizationQueryObjectType(IStringLocalizer<LocalizationQueryObjectType>
 
             Field(x => x.Culture).Description(S["The culture for your content item."]);
             Field(x => x.LocalizationSet).Description(S["The localization set for your content item."]);
+
+            Field<ListGraphType<ContentItemType>, IEnumerable<ContentItem>>()
+                .Name("Localizations")
+                .Description(S["The localizations of the content item."])
+                .Argument<StringGraphType, string>("culture", "the culture of the content item", string.Empty)
+                .Resolve(x =>
+                {
+                    var culture = x.GetArgument<string>("culture");
+
+                    var context = (GraphQLContext)x.UserContext;
+                    var contentLocalizationManager = context.ServiceProvider.GetService<IContentLocalizationManager>();
+
+                    if (!string.IsNullOrWhiteSpace(culture))
+                    {
+                        return new List<ContentItem>
+                        {
+                            contentLocalizationManager.GetContentItemAsync(x.Source.LocalizationSet, culture).GetAwaiter().GetResult()
```

Comment:
I'm not sure why you didn't use `await here?!!

Reply:
Hadn't realized there was a ResovleAsync method. Using that now and await